src/pages/ckbtc.tsx

```diff
@@ -538,7 +538,35 @@ function CkbtcPage(): JSX.Element {
 
                 <div>
                   <h3 className="tw-heading-5 md:tw-heading-3 mb-6">
-                    How it differs from Lightning.
+                    How ckBTC's security was assessed
+                  </h3>
+
+                  <p className="tw-paragraph mb-3">
+                    Several layers of security mesasures are taken on critical components of
+                    the Internet Computer such as ckBTC to ensure robust security. These
```

Comment:
```suggestion
                    Several security assessments are taken on critical components of
                    the Internet Computer such as ckBTC to ensure robust security. These
```
I think "security measures" refers more to design / code that prevents security risks vs. here we mean "we assessed if the code is secure" in teh examples